src/main/java/io/camunda/testing/assertions/ProcessInstanceAssertions.java

```diff
@@ -43,7 +43,8 @@ public ProcessInstanceAssertions isStarted() {
     final Optional<Record<ProcessInstanceRecordValue>> processInstanceRecord =
```

Comment:
Can you exclude this file from the PR?

Reply:
The changes in this file were part of the google java formatting. I think the new plugin has different rules from the one we used before. I do agree with these changes though, so If you don't mind I'd like to merge it with these in them. All the changes are is add new lines at chained method calls.

This is actually how I wrote it but the other plugin kept undoing it 😄